### ISSUE TITLE
Failed to launch hugo

### ISSUE THREAD

**Issue by darkdecoy** [mejedi/vscode-hugo-live-preview#4]
After installing version 0.1.0 and invoking 'Open Hugo Live preview' from the Command Palette to open a preview panel. I get the following error message.

```
Server unexpectedly terminated or failed to start

Failed to launch 'hugo': No such file or directory.

**Comment by mejedi** [comment]
@darkdecoy sorry for missing this one. This message means that Hugo executable file was not found. Are you able to start Hugo manually, eg from a terminal?

**Comment by darkdecoy** [comment]
Yes I can start from the terminal.  I think the issue was that hugo was installed as a snap and vscodium was installed as a flatpak
